src/main/io/serial_cli.c

```diff
@@ -2494,6 +2497,25 @@ static void cliMotor(char *cmdline)
     cliPrintf("motor %d: %d\r\n", motor_index, motor_disarmed[motor_index]);
 }
 
+static void cliName(char *cmdline)
+{
+    
+    uint32_t len = strlen(cmdline);
+    if (isEmpty(cmdline)) {
+        cliPrintf("name %s\r\n", masterConfig.name);
+    } else if (len <= MAX_NAME_LENGTH) {
+        strcpy(masterConfig.name, cmdline); 
+        for (uint8_t i = len; i<MAX_NAME_LENGTH; i++) {
+            masterConfig.name[i] = '\0';
+        }
+        cliPrintf("name %s\r\n", masterConfig.name);
+    } else {
+        cliPrintf("Max allowed name length is %d\r\n", MAX_NAME_LENGTH);
```

Comment:
Agreed. And I wouldn't bother printing the warning. Just print the truncated name.